src/Symfony/Component/Messenger/Tests/Transport/Serialization/PhpSerializerWithClassNotFoundSupportTest.php

```diff
@@ -22,6 +22,7 @@ class PhpSerializerWithClassNotFoundSupportTest extends PhpSerializerTest
     public function testDecodingFailsWithBadClass()
     {
         $this->expectException(MessageDecodingFailedException::class);
+        $this->expectExceptionMessageMatches('/Could not decode/');
```

Comment:
Please move the two expect*() calls before the `decode()` class

Reply:
Would you like me to fix all the test class at the same time?

Reply:
That would be awesome 👍 Thanks